lib/trace.js

```diff
@@ -0,0 +1,9 @@
+const { curry } = require('ramda');
```

Comment:
Again, I love that you built something to help make it easier to track down what was going on during your building process. While I noted the unused reference to this in your `index.js`, I'm personally ok with leaving this in the `lib` directory here as something you could use in the future.